src/pages/Overview/index.js

```diff
@@ -1,16 +1,880 @@
+    const { totalResultData } = this.state;
+    const updatedResult = totalResultData.filter(x => x.key !== row.key);
+    this.setState(
+      {
+        totalResultData: updatedResult,
+      },
+      () => {
+        this.getSeperatedResults();
+      }
+    );
+  };
+
+  handleTabClick = (event, tabIndex) => {
+    this.setState({
+      activeTabKey: tabIndex,
+    });
+  };
+
   render() {
+    const { activeTabKey, barWidth, newData, unlabledData } = this.state;
+
+    const newDataColumns = [
+      {
+        title: 'Result',
+        dataIndex: 'result',
+        key: 'result',
+        render: text => {
+          return (
+            <div>
+              <Button variant="link" isInline style={{ marginBottom: '8px' }}>
+                {text[0]}
+              </Button>
+              <br />
+              <Text component={TextVariants.p} className={styles.subText}>
+                <span className={styles.label}>{text[1]}</span>
+              </Text>
+            </div>
+          );
+        },
+      },
+      {
+        title: 'End Time',
+        dataIndex: 'end',
+        key: 'end',
+      },
+      {
+        title: '',
+        dataIndex: 'fav',
+        key: 'fav',
+        render: () => <Icon type="star" onClick={() => this.onFavourite()} />,
+      },
+      {
+        title: '',
+        dataIndex: 'action',
+        key: 'action',
+        render: (value, row) => {
+          return (
+            <div>
+              <EllipsisVIcon
+                onClick={() => this.showDrowpdown(`newrun${row.key}`)}
+                className="dropbtn"
+              />
+              <div className={styles.dropdown} id={`newrun${row.key}`} style={{ display: 'none' }}>
+                <div className={styles.dropdownContent}>
+                  <div className={styles.dropdownLink} onClick={() => this.markSeen(row)}>
+                    Mark as seen
+                  </div>
+                  <div className={styles.dropdownLink} onClick={() => this.deleteResult(row)}>
+                    Delete
+                  </div>
+                </div>
+              </div>
+            </div>
+          );
+        },
+      },
+    ];
+
+    const seenDataColumns = [
+      {
+        title: 'Result',
+        dataIndex: 'result',
+        key: 'result',
+        render: text => (
+          <div>
+            <Button variant="link" isInline style={{ marginBottom: '8px' }}>
+              {text[0]}
+            </Button>
+            <br />
+            <Text component={TextVariants.p} className={styles.subText}>
+              <span className={styles.label}>{text[1]}</span>
+            </Text>
+          </div>
+        ),
+      },
+      {
+        title: 'End Time',
+        dataIndex: 'end',
+        key: 'end',
+      },
+      {
+        title: 'Scheduled for deletion on',
+        dataIndex: 'deletion',
+        key: 'deletion',
+        render: text => {
+          const deleteDate = moment(new Date(Date.parse(text)));
```

Comment:
Is it possible to use the `getTimeDistance()` method in `utils.js` here?

Reply:
@gurbirkalsi as per my understanding, `getTimeDistance()` function is returning two moment objects having time distance of a year, month or days. Here we are trying to calculate the number of days remaining for the result deletion and the deletion date is being imported from mock value. I am not really sure how to incorporate `getTimeDistance()` function here.